### PR TITLE
Xen 4.16

### PR DESCRIPTION
**Pull request by jandryuk**
Policy updates for Xen 4.16.

This was on top of some usbvm work, so I left that in.

### CODE REVIEW

File: policy/flask/access_vectors [hunk 1]
```diff
@@ -67,10 +67,8 @@ class xen
     lockprof
 # XEN_SYSCTL_cpupool_op
     cpupool_op
-# tmem hypercall (any access)
```
Comment (crogers1) [inline policy/flask/access_vectors:70]: Found where this is dropped:
```
commit c492e19fdd05273417a1b116a90b26587738810c
Author: Wei Liu <wei.liu2@citrix.com>
Date:   Wed Nov 28 12:13:15 2018 +0000

    xen: remove tmem from hypervisor
```
was just double checking for it :+1:

File: policy/modules/xen/ndvm.te [hunk 2]
```diff
@@ -24,6 +24,7 @@ policy_module(ndvm, 0.2)
 #
 
 type ndvm_t;
+typealias ndvm_t alias { usbvm_t };
```
Comment (crogers1) [inline policy/modules/xen/ndvm.te:27]: leaving the usbvm stuff in here is fine, nothing is making use of it until we merge the main usbvm stuff.

File: policy/modules/xen/ndvm.te [hunk 3]
```diff
@@ -50,6 +51,7 @@ stubdom_event_src(ndvm_t, stubdom-ndvm_evchn_t)
 # Local Policy
 #
 
+dom0_ioemu(ndvm_t)
```
Comment (crogers1) [inline policy/modules/xen/ndvm.te:54]: What was the motivation for adding this rule?

Comment (jandryuk) [reply policy/modules/xen/ndvm.te:54]: I fixed up svirt-interpose and qemu running in dom0, and they need this.

**Comment by jandryuk** [comment]
I missed some Signed-off-by and will expand some commit messages.

File: policy/modules/xen/ndvm.te (hunk 3 above)
Comment (crogers1) [reply policy/modules/xen/ndvm.te:54]: Gotcha.  Thanks.

**Review by crogers1 — APPROVED** [review]
LGTM.  Queued up for merge when remainder of xen 4.16 PRs are reviewed.

**Pull request merged**